auth_supplier/models/res_users.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# (c) 2015 Antiun Ingeniería S.L. - Sergio Teruel
+# (c) 2015 Antiun Ingeniería S.L. - Carlos Dauden
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from openerp import models, api
+
+
+class ResUsers(models.Model):
+    _inherit = "res.users"
+
+    @api.model
+    def _signup_create_user(self, values):
+        res = super(ResUsers, self)._signup_create_user(values)
+        if isinstance(res, int):
+            user = self.env['res.users'].browse(res)
+            account_type = values.get('account_type', False)
+            if account_type and account_type == 'supplier':
```

Comment:
Probably because of this, this module just does not work (in runbot at least):

![captura de pantalla de 2015-10-20 16-30-58](https://cloud.githubusercontent.com/assets/973709/10610437/11b23a42-7748-11e5-81fa-8624475b0062.png)